lib/ui/semantics/semantics_node.h

```diff
@@ -57,6 +57,19 @@ const int kHorizontalScrollSemanticsActions =
 const int kScrollableSemanticsActions =
     kVerticalScrollSemanticsActions | kHorizontalScrollSemanticsActions;
 
+/// C/C++ representation of `SemanticsRole` defined in
+/// `lib/ui/semantics.dart`.
+///\warning This must match the `SemanticsRole` enum in
+///         `lib/ui/semantics.dart`.
+/// See also:
+///   - file://./../../../lib/ui/semantics.dart
+enum class SemanticsRole : int32_t {
+  kNone = 0,
+  kTab = 1,
+  kTabBar = 2,
+  kTabPanel = 3,
+};
```

Comment:
I assume a C version of this change will also need to be added embedder.h?

Reply:
I will do that in a separate pr when i add the support to the desktop embedding. I am trying to avoid adding support for all platforms in a single pr